recipes/repository.rb

```diff
@@ -71,15 +79,20 @@
     end
   end
 
+  case node['datadog']['agent_major_version'].to_i
```

Comment:
This is broken on CentOS (the package is not found in the repos), I think that's because the `baseurl` always need to be computed but this is not working if `agent_major_version` is not set.
Also, it should have to take care of an inconsistent `agent_version` with a major different that `agent_major_version`.

Reply:
`agent_major_version` should always be set.

Reply:
You're right it's properly set in the default attributes, the problem must be elsewhere then 🤔

Reply:
Fixed.